lib/services/runtime/handlers/noMatch.ts

```diff
@@ -10,20 +10,28 @@ export type NoMatchCounterStorage = number;
 
 export const EMPTY_AUDIO_STRING = '<audio src=""/>';
 
-const removeEmptyNoMatches = (noMatchArray?: string[]) => noMatchArray?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+type NoMatchNode = BaseNode.Utils.BaseNode & VoiceNode.Utils.DeprecatedNodeNoMatch & { noMatch?: Nullable<VoiceNode.Utils.NodeNoMatch> };
+
+const removeEmptyNoMatches = (node: NoMatchNode) => {
+  const noMatches = node.noMatches || node.noMatch?.noMatches || [];
+  return noMatches?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+};
```

Comment:
I guess we should first check the new data and fallback onto the old one

```suggestion
const removeEmptyNoMatches = (node: VoiceNode.Utils.NoMatchNode) => {
  const noMatches = node.noMatch?.prompts ?? node.noMatches ?? [];
  
  return noMatches.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
};
```